index.html

```diff
@@ -35,6 +35,9 @@ <h3 id="head">Here's a time line of Dr. Borlaug's life:</h3>
             <strong>1933 </strong>  1933 - Leaves his family's farm to attend 
             the University of Minnesota, thanks to a Depression era program known as the
              "National Youth Administration" 
+
+             <hr>
+             i remembered to make some changes here
```

Comment:
### Useless code
remember to remove the useless code. to make your code clean.we clean this and it's good
![SpongeBobCleaningGIF](https://user-images.githubusercontent.com/40032163/165135482-fd886ace-470c-4bc9-8ffe-cb08450b09da.gif)

Reply:
- [ ] **[REQUIRED CHANGES]**
Great job on your review but i think you're missing something which is the main objective of making a code review.
- You have to show sympathy and put yourself in the student's shoes. You saying `**useless code **` sounds so disheartning as your tone presently can make a student feel embarrassed. You should use polite tones..
> you can say `I think leaving unused code.............. [state your reasons]`. Please, kindly update your response

Reply:
Please try to modify your review as the previous reviewer asked by making your comment more gentile try to use polite phrases like: `it would be nice if you always  keep your files clean by removing not used code that will keep your repo pro`